### PR TITLE
Add --export to load functions

### PR DESCRIPTION
**Pull request by jmaslek**
As the title says, this adds export to the various loads.  The reason is that this makes life easier to use the econometrics and forecasting menu.

Now we can have a script that would do

```
stocks
load aapl --export aapl.csv
load msft --export msft.csv
load amzn --export amzn.csv
econometrics
....
```

### CODE REVIEW

File: openbb_terminal/parent_classes.py [hunk 1]
```diff
@@ -1181,3 +1192,9 @@ def call_load(self, other_args):
                     ns_parser.exchange,
                     self.current_interval,
                 )
+                export_data(
+                    ns_parser.export,
+                    os.path.dirname(os.path.abspath(__file__)),
+                    "load",
+                    self.current_df.copy(),
```
Comment (DidierRLopes) [inline openbb_terminal/parent_classes.py:1199]: Why `copy()` here?

Comment (jmaslek) [reply openbb_terminal/parent_classes.py:1199]: i don't want to goof up the dataframe lol

**Review by DidierRLopes — APPROVED** [review]

File: openbb_terminal/parent_classes.py (hunk 1 above)
Comment (DidierRLopes) [reply openbb_terminal/parent_classes.py:1199]: What do you mean by goof up? At that stage we just save it, there's nothing happening with it after.

Comment (jmaslek) [reply openbb_terminal/parent_classes.py:1199]: I had an edge case in my brain at the time but can't think of what it is now

**Review by martinb-bb — APPROVED** [review]

**Pull request merged**